src/pages/Home/components/HomeHeader/HomeHeader.vue

```diff
@@ -69,7 +67,7 @@ export interface HomeHeaderProps {
 const props = defineProps<HomeHeaderProps>();
 
 const userMembershipStatus = computed(() =>
-  props.isMembershipActive ? 'Usuario activo' : 'Usuario inactivo'
+  props.isMembershipActive ? 'Usuario' : 'Usuario'
```

Comment:
I don't like this change, please check

Reply:
@angelik0828

Reply:
@victor-heliomar please check

Reply:
@Eghinner now there is not a membership? Maybe remove this variable and just hardcode it in the code